packages/odyssey-react/src/components/Toast/Toast.tsx

```diff
@@ -144,7 +145,7 @@ export const Toast = withTheme(
         color={false}
       >
         <span className={styles.icon}>{icon[variant]}</span>
-        <h1 className={styles.title}>{title}</h1>
+        <h1 className={styles.heading}>{title}</h1>
```

Comment:
Does the story need an update to account for this?

Reply:
This particular change only touches the styles, I figured leave the prop / api change for the other story